lib/mayaUsd/python/wrapOpUndoItem.cpp

```diff
@@ -59,6 +59,18 @@ class PythonOpUndoItemList
         _undoItemList.reset();
     }
 
+    void undo()
+    {
+        if (_undoItemList)
```

Comment:
In enter() and exit() you have this check as a TF_VERIFY, should do the same here, no?